lib/v2/screens/app/interactor/viewmodels/app_event.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+
+/// --- EVENTS
+@immutable
+abstract class AppEvent extends Equatable {
+  const AppEvent();
+  @override
+  List<Object> get props => [];
+}
+
+class ShowNotificationBadge extends AppEvent {
+  final bool value;
```

Comment:
mark this as required and set initial to false

Reply:
and give it a better name `shouldShowNotificationBadge`

then when used inside an if reads better 

if(shouldShowNotificationBadge) else

Reply:
done 👍